kvrocks.conf

```diff
@@ -43,8 +43,11 @@ daemonize no
 # If you enable cluster, kvrocks will encode key with its slot id calculated by
 # CRC16 and modulo 16384, encoding key with its slot id makes it efficient to
 # migrate keys based on the slot. So if you enabled at first time, cluster mode must
-# not be disabled after restarting, and vice versa. That is to say, data is not
-# compatible between standalone mode with cluster mode, you must migrate data
+# not be disabled after restarting, and vice versa. Currently, kvrocks will keep 
+# using the cluster-enabled status that is persisted at first time, regardless of 
+# what cluster-enabled status is provided afterwards.
```

Comment:
force rewrite or can not start the server? HDYT @git-hulk @PragmaTwice

Reply:
Both of them are good for me. I prefer throwing an error if we have to choose one.

Reply:
Okay I'll change accordingly. By throwing an error you mean also to prevent the user from starting the server on mismatch right?

Reply:
Me too, I also tend to just throw an error that prevents the user from starting the server. This is equivalent to adding a quick pre check.

Reply:
> By throwing an error you mean also to prevent the user from starting the server on mismatch right?

That's right.